manuscript/teaching-reproducibility.Rmd

```diff
@@ -141,7 +141,7 @@ in classrooms of 10-200 learners from varying educational backgrounds,
 including undergraduate students taking their first data science course
 and graduate students looking to specialize in data science.
 We teach the plurality of our courses in the Master of Data Science program,
-which consists of 24 one-credit courses
+which consists of 24 one-credit courses over an eight month period
```

Comment:
Added this here and removed the equivalent sentence in the reproducibility section.